extras/container/heketi-start.sh

```diff
@@ -91,7 +159,7 @@ fi
 # heketi service back to the foreground again.
 if [[ "$(stat -c %s ${HEKETI_PATH}/heketi.db)" == 0 && -n "${HEKETI_TOPOLOGY_FILE}" ]]; then
     # start hketi in the background
-    /usr/bin/heketi --config=/etc/heketi/heketi.json &
+    "$HEKETI_BIN" --config=/etc/heketi/heketi.json &
```

Comment:
This looks weird. Why not just do it like

    ${HEKETI_BIN} --config=/etc/heketi/heketi.json &

Or is there really an expectation that "HEKETI_BIN" may not have any spaces? Maybe it would be useful to be able to pass additional options that way...

Reply:
Force of habit mostly.

Reply:
Well, in theory the path to heketi bin could contain spaces, then you need the double quotes.